src/com/tutorial/main/GameObject.java

```diff
@@ -1,55 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Graphics;
-import java.awt.Rectangle;
-
-public abstract class GameObject {
```

Comment:
That's pretty much it.

`GameObject` would be considered a superclass, and then `BasicEnemy` would be a subclass.

You could do `new GameObject()` in what is called an anonymous inner class. Would look something like:
````
GameObject anonymousGameObject = new GameObject(0, 0, null) {
    public void tick() {
    }
    public void render(Graphics g) {
    }
    public Rectangle getBounds() {
        return null;
    }
}
````